GitCommands/RevisionReader.cs

```diff
@@ -516,7 +494,7 @@ int CountParents(in ReadOnlySpan<byte> array, int baseOffset)
 
             if (_hasNotes)
             {
-                revision.HasNotes = true;
+                revision.Notes ??= "";
             }
```

Comment:
If needed add after line 491. 
(OK to be null if notes are fetched?)

Reply:
Not sure what do you mean.

Reply:
I should have written _assign to Notes_ needed?
Default is null anyway.

Reply:
Looks like HasNotes true/false effectively changed into Notes null/nonNull.